solution.py

```diff
@@ -1378,19 +1605,31 @@ def min_max_scale(image:ArrayLike)->ArrayLike:
 
         test_segmentation_metrics.loc[len(test_segmentation_metrics)] = {
             "model": "phase2fluor",
-            "fov":fov,
+            "fov": fov,
             "masks_per_fov": num_masks_fov,
-            "accuracy": accuracy(pred_label_binary, target_label_binary, task="binary").item(),
-            "dice":  dice(pred_label_binary, target_label_binary).item(),
-            "jaccard": jaccard_index(pred_label_binary, target_label_binary, task="binary").item(),
-            "mAP":coco_metrics["map"].item(),
-            "mAP_50":coco_metrics["map_50"].item(),
-            "mAP_75":coco_metrics["map_75"].item(),
-            "mAR_100":coco_metrics["mar_100"].item()
+            "accuracy": accuracy(
+                pred_label_binary, target_label_binary, task="binary"
+            ).item(),
+            "dice": dice_score(
+                pred_label_binary.long()[None],
+                target_label_binary.long()[None],
+                num_classes=2,
+                input_format="index",
+                average="micro",
+            ).item(),
+            "jaccard": jaccard_index(
+                pred_label_binary, target_label_binary, task="binary"
+            ).item(),
+            "mAP": coco_metrics["map"].item(),
+            "mAP_50": coco_metrics["map_50"].item(),
+            "mAP_75": coco_metrics["map_75"].item(),
+            "mAR_100": coco_metrics["mar_100"].item(),
         }
 
-        pred_label,target_label= cellpose_segmentation(predicted_nuc_pretrained,target_nucleus)
-        
+        pred_label, target_label = cellpose_segmentation(
+            predicted_nuc_pretrained, target_nucleus
```

Comment:
```suggestion
            predicted_nuc_pretrained, target_nucleus[0]
```
`target_nucleus` is 4-dimensional here (`(1,1,2048,2048)`), which tricks Cellpose into thinking this is a 3D stack and an error is raised.